rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMRest.java

```diff
@@ -269,7 +269,7 @@ public RMState getState(@HeaderParam("sessionid") String sessionId) throws NotCo
     @GZIP
     @Path("monitoring")
     @Produces("application/json")
-    public RMInitialState getInitialState(@HeaderParam("sessionid") String sessionId,
+    public RMStateDTO getInitialState(@HeaderParam("sessionid") String sessionId,
```

Comment:
What does DTO mean?

Reply:
Data Transfer Object, 
i will clarify in the code as well

Reply:
Maybe RMStateDelta ?

Reply:
Ok